File: app/src/main/java/com/gnoemes/shikimoriapp/entity/anime/domain/AnimeStatus.java
package com.gnoemes.shikimoriapp.entity.anime.domain;

public enum AnimeStatus {
    ANONS("anons"),
    ONGOING("ongoing"),
    RELEASED("released"),
    NONE("none");


    private final String status;

    AnimeStatus(String status) {
        this.status = status;
    }

    public boolean equalsStatus(String otherStatus) {
        return status.equals(otherStatus);
    }

    @Override
    public String toString() {
        return status;
    }
}

File: app/src/main/java/com/gnoemes/shikimoriapp/entity/anime/domain/AnimeStatus.kt
package com.gnoemes.shikimoriapp.entity.anime.domain

import com.google.gson.annotations.SerializedName

enum class AnimeStatus {
    @SerializedName("anons")
    ANONS,
    @SerializedName("ongoing")
    ONGOING,
    @SerializedName("released")
    RELEASED,
    @SerializedName("")
    NONE
}